region/lowernorfair/east.json

```diff
@@ -812,9 +909,30 @@
           "nodeSubType": "blue",
           "sparking": {
             "runways": [
+              {
+                "length": 15,
+                "requires": [
+                  {"or": [
+                    "heatProof",
+                    "canHeatRun"
+                  ]},
+                  {"cost": [
+                    {"heatFrames": 175}
+                  ]}
+                ]
+              },
               {
                 "length": 19,
-                "requires": null
+                "requires": [
+                  {"or": [
+                    "heatProof",
+                    "canHeatRun"
+                  ]},
+                  "Super",
```

Comment:
Why specifically Super? There's no other good way to kill it?

Reply:
Also, there should probably a way to use this by killing the pirate by any slower means (except maybe morph bombs)? We could have it require heatproof at worst.

Reply:
If you use the fast Supers you can escape an extra cycle of acid rising onto the runway but I guess the cycle isn't that extremely long so I'm including the slower alternatives of Missiles and Charge+Plasma.